content/en/docs/tasks/tools/_index.md

```diff
@@ -2,5 +2,40 @@
 title: "Install Tools"
 description: Set up Kubernetes tools on your computer.
 weight: 10
+no_list: true
 ---
 
+## kubectl
+
+The Kubernetes command-line tool, `kubectl`, allows you to run commands against
+Kubernetes clusters. You can use kubectl to deploy applications, inspect and manage
+cluster resources, and view logs.
+
+[Install and Set Up kubectl](/docs/tasks/tools/install-kubectl/) explains how to
+download and install `kubectl` and set it up for accessing your cluster.
+
+You can also read the [`kubectl` reference documentation](/docs/reference/kubectl/).
+
+## Minikube
+
+[Minikube](https://minikube.sigs.k8s.io/) is a tool that makes it easy to run
+Kubernetes locally. Minikube runs a single-node Kubernetes cluster on your personal
+computer (including Windows, macOS and Linux PCs) so that you can try out Kubernetes,
+or for daily development work.
+
+You can follow the official [Get Started!](https://minikube.sigs.k8s.io/docs/start/)
+guide, or read [Install Minikube](/docs/tasks/tools/install-minikube/) if your focus
+is on getting the tool installed.
+
+Once you have Minikube working, you can use it to
+[run a sample application](/docs/tutorials/hello-minikube/).
+
+## kind
+
+[kind](https://kind.sigs.k8s.io/docs/) is another tool for running Kubernetes locally.
```

Comment:
Same here for starting a sentence with a link, to your satisfaction, etc. Perhaps:
```suggestion
kind is [another tool](https://kind.sigs.k8s.io/docs/) for running Kubernetes locally.
```